packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -146,11 +146,9 @@ export class AtomicSegmentedFacet
   }
 
   public render() {
-    return (
-      <div class="flex">
-        <b class="inline-block my-3 mr-2">{this.label}</b>
-        {this.renderValues()}
-      </div>
-    );
+    return [
+      <b class="inline-block my-3 mr-2">{this.label}</b>,
+      this.renderValues(),
+    ];
```

Comment:
Why this change?

Reply:
Still haven't merged in the newest branch